lib/gems/pending/appliance_console/database_replication_standby.rb

```diff
@@ -35,6 +37,25 @@ def ask_questions
       confirm
     end
 
+    def choose_disk
+      @disk = ask_for_disk("database disk")
+    end
+
+    def initialize_postgresql_disk
```

Comment:
I was thinking that too, but `LogicalVolumeManagement` is internal to appliance console. It would be odd for `PostgresAdmin` to require into the appliance_console files.